src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA_SYMBOL = ",";
+    private static final int CSV_VALUE_INDEX = 1;
+    private static final int CSV_KEY_INDEX = 0;
```

Comment:
dont invent your own names
```suggestion
    private static final int AMOUNT_INDEX = 0;
    private static final int OPERATION_TYPE_INDEX = 1;
```
![image](https://github.com/mate-academy/jv-work-with-file/assets/73041091/c40a7ca5-168f-409d-a875-5a8381cdc456)